fastlane/jetpack_metadata/android/de-DE/title.txt

```diff
@@ -1 +1 @@
-Jetpack: WP-Sicherheit und -Geschwindigkeit
+Jetpack
```

Comment:
All those changes in `title.txt` and `short_description.txt` for Jetpack are related to pxLjZ-6Vi-p2#comment-18922